exercises/linked-list/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "linked-list"
+version = "0.1.0"
+authors = ["Emerentius <emerentius@arcor.de>"]
```

Comment:
Strip out the authors line, please; that's a track policy we came to some time ago. I can track down the discussion if you like.